app/mailers/ladder_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class LadderMailer < ActionMailer::Base
+  default from: "engsci.website@gmail.com"
```

Comment:
Due to the auto emailler and auto responses, I figured it was much faster to do all the website stuff within its own email compared to the engsci@g.skule.ca email. Also I didn't want to inundate the exec with emails.